docs/build/isc/v1.0.0-rc.6/docs/schema/how-tos/state.mdx

```diff
@@ -173,7 +173,7 @@ export class MutableDividendState extends wasmtypes.ScProxy {
 
 Examining the `state.xx`code, generated by the [Schema Tool](usage.mdx),
 you can find the `MutableDividendState` struct.
-This interface allows type-safe access to each state variable through mutable [proxies](../../explanations/proxies.mdx),
+This interface allows type-safe access to each state variable through mutable [proxies](/isc/explanations/proxies),
```

Comment:
Why is this still not a file path?

Reply:
https://github.com/iotaledger/iota-wiki/pull/1415/commits/67b4cecd33c24aa1152ce413f4cb550bf0ae4dc5